include/worker.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright © 2017 Drew Walters <drewwalters96@gmail.com>
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#ifndef WORKER_H
+#define WORKER_H
+
+/* Queue worker function capacity. */
+#define MAX_WORKERS 1
```

Comment:
This is an implementation detail, so it belongs in the C file.

Reply:
Fixed